html/host.html

```diff
@@ -3,66 +3,100 @@
     <meta charset="utf-8"/>
     <link href="/assets/stylesheets/main.css" rel="stylesheet">
     <link href="/assets/stylesheets/host.css" rel="stylesheet">
+    <script type="module">
+      import { 
+        fullActivityDescription, 
+        shortActivityDescription, 
+        libraryModelList, 
+        cclDiseaseActivityUrl 
+      } from '/assets/data/host.js';
+
+      document.addEventListener("DOMContentLoaded", () => {
+        // Host page content init
+        document.getElementById("activity-text-full").innerHTML = fullActivityDescription;
+        document.getElementById("activity-text-short").innerHTML = shortActivityDescription;
+        document.getElementById("library-model").innerHTML = libraryModelList;
+        document.getElementById("activity-img").src = cclDiseaseActivityUrl;
+
+        // Open & close modal functionality
+        const readMoreBtn = document.getElementById("read-more-btn");
+        const closeModalBtn = document.getElementById("close-modal-btn");
+        const modalContainer = document.getElementById("modal-container");
+
+        readMoreBtn.onclick = () => {
+          modalContainer.style.display = "block";
+        };
+
+        closeModalBtn.onclick = () => {
+          modalContainer.style.display = "none";
+        };
+
+        window.onclick = (event) => {
+          if (event.target == modalContainer) {
+            modalContainer.style.display = "none";
+          }
+        };
+
+        document.addEventListener("keydown", (event) => {
+          if (modalContainer.style.display === "block" && event.key === "Escape") {
+            modalContainer.style.display = "none";
+          }
+        });
+      });
+    </script>
   </head>
   <body>
 
+    <!-- Modal container with full activity description (entire screen)-->
+    <div id="modal-container" class="modal-container">
+
+      <!-- Actual modal here -->
+      <div id="full-description-modal" class="modal">
+        <div class="modal-text">
+          <p class="modal-activity-title">Disease</p>
+          <p id="activity-text-full" class="activity-text-full"></p>
+        </div>
+        <hr class="modal-delineation"></hr>
+        <button id="close-modal-btn" class="close-modal-btn" type="button">Close</button>
+      </div>
+      
+    </div>
+  
     <div id="form-frame" class="frame">
-      <h1>You're a hoster</h1>
+      <p class="host-header">Hosting</p>
```

Comment:
What's the reasoning for replacing the perfectly semantic `h1` tag with a `p` tag?

Reply:
It was originally because the `<h1>` tag enforces certain styling that looked a bit weird (font size a bit too small, too much spacing, etc.), but since I ended up styling the `<p>` tag anyways, I just changed this back to an `<h1>` and used the same styling, so everything still looks the same. But now with an `<h1>`, not a `<p>`.